tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "outDir": "./dist/out-tsc",
     "sourceMap": true,
     "declaration": false,
-    "module": "es2015",
+    "module": "esnext",
```

Comment:
Latest and greatest?

Reply:
updated with the auto `ng update` migration